packages/inferno-router/__tests__/router.spec.jsx

```diff
@@ -91,6 +91,17 @@ describe('Router (jsx)', () => {
 			);
 			expect(container.innerHTML).to.equal(innerHTML('<div><p>Parent Component</p><div>Child is bar</div></div>'));
 		});
+		it('should render the child and inherit parent (URL containing percent encoded value)', () => {
+			render(
+				<Router url={ '/foo/100%' } history={ browserHistory }>
```

Comment:
I changed this to be percent decoded since much of the code expects unencoded values for the url.